eventmesh-connector-plugin/eventmesh-connector-mongodb/src/main/java/org/apache/eventmesh/connector/mongodb/producer/MongodbReplicaSetProducer.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.eventmesh.connector.mongodb.producer;
+
+import org.apache.eventmesh.api.RequestReplyCallback;
+import org.apache.eventmesh.api.SendCallback;
+import org.apache.eventmesh.api.SendResult;
+import org.apache.eventmesh.api.exception.ConnectorRuntimeException;
+import org.apache.eventmesh.api.exception.OnExceptionContext;
+import org.apache.eventmesh.api.producer.Producer;
+import org.apache.eventmesh.connector.mongodb.client.MongodbClientManager;
+import org.apache.eventmesh.connector.mongodb.config.ConfigurationHolder;
+import org.apache.eventmesh.connector.mongodb.utils.MongodbCloudEventUtil;
+
+import java.util.Properties;
+
+import org.bson.Document;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.cloudevents.CloudEvent;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoCollection;
+
+public class MongodbReplicaSetProducer implements Producer {
+    private static final Logger logger = LoggerFactory.getLogger(MongodbReplicaSetProducer.class);
```

Comment:
How about use lombok annotation @Slf4j

Reply:
@xwm1992 is there such a requirement for use?  I'll modify it if there is.

Reply:
> @xwm1992 is there such a requirement for use? I'll modify it if there is.

Master branch that I have done this.